Makefile

```diff
@@ -62,6 +62,16 @@ E2E_SKIP_CONTAINER_PUSH?=false
 # 	make e2e E2E_GO_TEST_FLAGS="-v -run TestE2E/TestScanWithNodeSelectorFiltersCorrectly"
 E2E_GO_TEST_FLAGS?=-v -timeout 120m
 
+# operator-courier arguments for `make publish`.
+# Before running `make publish`, install operator-courier with `pip3 install operator-courier` and create
+# ~/.quay containing your quay.io token.
+COURIER_CMD=operator-courier
+COURIER_PACKAGE_NAME=compliance-operator-bundle
+COURIER_OPERATOR_DIR=deploy/olm-catalog/compliance-operator
+COURIER_QUAY_NAMESPACE=compliance-operator
+COURIER_PACKAGE_VERSION="0.1.0"
```

Comment:
lets make this configurable (I think you need to do `COURIER_PACKAGE_VERSION?="0.1.0"` )